packages/ui-components/src/components/calendar-advanced-date-selector/calendar-advanced-date-selector.helper.ts

```diff
@@ -48,25 +49,30 @@ export const buildTimezonesDropdownOptions = (timezones: ITimezoneOffset[]): ISe
 
 	let otherTimezoneGroups: ISelectSingleOptions = {};
 	if (!isEmpty(timezones)) {
-		otherTimezoneGroups = {
-			[OTHER_TIMEZONES_GROUP_NAME]: {
-				label: OTHER_TIMEZONES_GROUP_LABEL,
-				value: OTHER_TIMEZONES_GROUP_NAME,
-				options: filterDefaultTimezone(timezones, timezone?.name).reduce<ISelectSingleOptions>((accumulator, { label, name }) => {
-					accumulator[name] = accumulator[name] ?? buildDropdownOption(label, name);
-
-					return accumulator;
-				}, {})
-			}
-		};
+		const timezonesWithoutDefault = filterDefaultTimezone(timezones, timezone?.name);
+		const filteredTimezoneOptions = timezonesWithoutDefault.reduce<ISelectSingleOptions>((accumulator, { label, name }) => {
+			accumulator[name] = accumulator[name] ?? buildDropdownOption(label, name);
+
+			return accumulator;
+		}, {});
+
+		otherTimezoneGroups =
```

Comment:
@djcouto I am also applying this logic here because even though the Other Timezones option is no longer selectable, It was still showing up because we consider the parent option as if it was an option. Since we dont want this option to appear empty, I dont send it if no options are available.

Reply:
Reverted this changes